integration

```diff
@@ -237,4 +238,4 @@
   "xMrVizzy/Philips-AirPurifier",
   "zachowj/hass-node-red",
   "zha-ng/zha-map"
-]
\ No newline at end of file
+]
```

Comment:
I made the PR from GitHub UI and this was removed by GitHub online editor. Is it a problem?

Reply:
no